### ISSUE TITLE
Need environment.yml or requirements.txt for reproducible setup

### ISSUE THREAD

**Issue by emircetinmemis** [SiTH-Diffusion/SiTH#22]
Running into environment setup issues:

## Current Problems
- Package conflicts such like numpy/torch versions or other undefined packages.
- Unclear used Python version.

## Requested Information
- Used Python version
- environment.yml or at least pip freezed requirements.txt with working package versions
- Setup instructions for common environments (conda/pip)

## Example Errors
```python
# NumPy compatibility issue
ValueError: numpy.dtype size changed, may indicate binary incompatibility...

# Diffusers/HuggingFace conflict
ImportError: cannot import name 'cached_download' from 'huggingface_hub'
```

**Comment by azuxmioy** [comment]
This project was implemented with Python 3.8.10.

For the errors, please consider downgrading the packages:
```
numpy==1.24.1 
huggingface-hub==0.22.2
```

**Comment by emircetinmemis** [comment]
Thanks for providing the Python version and package details.

I will be trying them soon, gonna update you about it also.

Additionally, it would be really nice if you could share a `pip freeze > requirements.txt` resulting file, but if it is not available right now, I'm OK with that.

Thanks in advance
